assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -153,6 +166,62 @@ export const CookiesFrontend = props => {
         />
       </Fragment>
       }
+      {((ENABLE_ANALYTICAL_COOKIES) && (isEditing || (analytical_cookies_name && analytical_cookies_description))) &&
+      <Fragment>
+        <label className="custom-control"
+              style={isSelected ? { pointerEvents: 'none' } : null}>
+          <input
+            type="checkbox"
+            tabIndex={isSelected ? '-1' : null}
+            name="analytical_cookies"
+            onChange={ () => {
+              if (analyticalCookiesChecked) {
+                setUserRevokedAnalytical(true);
+                setUserRevokedAllCookies(true);
+                if (allCookiesChecked) {
+                  setConsentCookie(ALL_COOKIES);
+                } else {
+                  setConsentCookie(ONLY_NECESSARY);
+                }
+              } else {
+                if (allCookiesChecked) {
+                  setConsentCookie(NECESSARY_ANALYTICAL_MARKETING);
+                } else {
+                  setConsentCookie(NECESSARY_ANALYTICAL);
+                }
+              }
+            } }
+            checked={analyticalCookiesChecked}
+            className="p4-custom-control-input"
+          />
+          <FrontendRichText
+            tagName="span"
+            className="custom-control-description"
+            placeholder={__('Enter analytical cookies name', 'planet4-blocks-backend')}
+            value={analytical_cookies_name}
+            onChange={toAttribute('analytical_cookies_name')}
+            keepPlaceholderOnFocus={true}
+            withoutInteractiveFormatting
+            characterLimit={40}
```

Comment:
This prop doesn't exist for this component, so it doesn't do anything, and causes warnings in the console. I created a PR to remove similar usages that were already merged https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/654/files